code/controllers/subsystem/statistics.dm

```diff
@@ -0,0 +1,7 @@
+//This isn't actually a subsystem. It exists solely to hold a game's round statistics, such as deaths.
```

Comment:
just put it as a global var datum initialised at world start or attach it to world var